### PR TITLE
Added weppy to python frameworks

### PR DESCRIPTION
**Pull request by gi0baro**
This adds [weppy](http://weppy.org) framework to python benchmarks.

Tests included:
* JSON Serialization
* Single Database Query
* Multiple Database Queries
* Fortunes
* Database Updates
* Plaintext

Environments matrix:
* py2
* py3
* pypy
* nginx-uwsgi

### CODE REVIEW

**Comment by knewmanTE** [comment]
Thanks for your merge request @gi0baro! For our round 13 test, we're currently only accepting bug fixes and pull requests that directly address what we're seeing the results of the round 13 preview test. All new frameworks or larger changes will be included in round 14. Would you mind closing this merge request and opening up a new one against the round-14 branch?

**Comment by gi0baro** [comment]
@knewmanTE sure, sorry for the inconvenient.

**Comment by knewmanTE** [comment]
@gdamjan no worries! I realize the branching system is a bit confusing. Thanks for being so quick to respond!

File: frameworks/Python/weppy/uwsgi.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[uwsgi]
+master
+; Increase listen queue used for nginx connecting to uWSGI. This matches
+; net.ipv4.tcp_max_syn_backlog and net.core.somaxconn.
+listen = 65535
```
Comment (methane) [inline frameworks/Python/weppy/uwsgi.ini]: Please use 128.  uWSGI fails to start when net.core.somaxconn is not large enough.

Comment (gi0baro) [inline frameworks/Python/weppy/uwsgi.ini]: @methane the correct PR is #2228 
I copied that from the flask example: https://github.com/TechEmpower/FrameworkBenchmarks/blob/round-14/frameworks/Python/flask/uwsgi.ini#L5

If this value is not correct then should be changed on every framework, not just this.

Comment (methane) [inline frameworks/Python/weppy/uwsgi.ini]: > the correct PR is #2228

Sorry.

> If this value is not correct then should be changed on every framework, not just this.

Yes.  All tests using uwsgi failed on Round 13 Preview.
I've sent pull request already to fix Round 13 final (#2227).

Comment (gi0baro) [inline frameworks/Python/weppy/uwsgi.ini]: @methane Ok, I missed it. Will patch #2228, thank you!

**Pull request closed without merging**